.eslintrc.js

```diff
@@ -1,4 +1,5 @@
 module.exports = {
   root: true,
+  parser: 'babel-eslint',
```

Comment:
I don't think this is actually necessary. Can you revert this and verify using `yarn lint:js` instead of using the IDE-integrated linter?

Reply:
Sure, this is what ember-decorator's blueprint did, so I assumed it was needed. Sorry I'm not familiar with all this stuff .